src/lib/server.ts

```diff
@@ -26,10 +26,11 @@ export const createServer = async (): Promise<Server> => {
 	await populateIpWhiteList();
 	// Populate cities info
 	await populateCitiesList();
+	// Populate Dashboard override data
+	await probeOverride.fetchDashboardData();
```

Comment:
That change is made to fetch override data before using it during `initWsServer()`:
https://github.com/jsdelivr/globalping/blob/master/src/lib/ws/synced-probe-list.ts#L185-L186